src/QMCWaveFunctions/lcao/RadialOrbitalSetBuilder.h

```diff
@@ -177,9 +172,9 @@ class RadialOrbitalSetBuilder: public MPIObjectBase
 };
 
   template<typename COT>
-    RadialOrbitalSetBuilder<COT>::RadialOrbitalSetBuilder(Communicate* comm)
+  RadialOrbitalSetBuilder<COT>::RadialOrbitalSetBuilder(Communicate* comm, int radial_grid_size)
```

Comment:
Could you explain a bit how radial_grid_size is handled after the change?
This maybe affects the input of grid.

Reply:
Now it is a magic number in one place.  It will be easier to work on making it something more optimized than just 1001 typed into the source in multiple places.